README.md

```diff
@@ -0,0 +1,65 @@
+# C4C Dev Docs
+
+C4C Dev Docs holds all technical documentation for our applications and projects. The goal of this site is to bring existing information together to a single, searchable, and pleasant UI. 
+
+As of now, we've copied over the LLB and SFTT API specs. There is some work to update those to be more user-friendly for this site, but I'm very happy with how MkDocs took our existing Markdown files and created this site so effortlessly. 
+
+Other things to expect to see here: general repo setup guide for our major projects, DevOps/Deployment overview for LLB/SFTT cloud pipelines, Jumpstart course material...
```

Comment:
This is probably small but instead of saying repo we should say that repo is short for repository beforehand or explain it here.